app/models/provider_ucas_preference.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: provider_ucas_preference
+#
+#  id                      :bigint(8)        not null, primary key
+#  provider_id             :integer          not null
+#  type_of_gt12            :text
+#  send_application_alerts :text
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#
+
+class ProviderUCASPreference < ApplicationRecord
```

Comment:
Can this just be `UCASPreference` #namingishard

Reply:
I wanted the table and model to be clearly linked to the provider. The relationship strips the `provider_` part off, since it's redundant when applied to a provider there.

What do others think?